nixos/modules/services/video/frigate.nix

```diff
@@ -271,7 +271,7 @@ in
               sub_filter '"/BASE_PATH/monacoeditorwork/' '"$http_x_ingress_path/assets/';
               sub_filter 'return"/BASE_PATH/"' 'return window.baseUrl';
               sub_filter '<body>' '<body><script>window.baseUrl="$http_x_ingress_path/";</script>';
-              sub_filter_types text/css application/javascript;
+              sub_filter_types text/css text/javascript;
```

Comment:
Hold on, why was this mime type changed? It doesn't look related to mailcap?

Reply:
mailcap, which is a dependency of nginx, changed the MIME type of .js files from `application/javascript` to `text/javascript` (see the diff linked in the PR description). So I changed the usages of the old mime type to match.

However, I looked at this half a year ago. There may have been more usages added since then. I'd have to revisit this update, and I don't have time currently. Closing this for now.

Reply:
> mailcap, **which is a dependency of nginx**

I didn't know that :innocent:. The changes make more sense then.

> However, I looked at this half a year ago. There may have been more usages added since then. I'd have to revisit this update, and I don't have time currently. Closing this for now.

Understandable.